src/kirby/components/page/page.component.scss

```diff
@@ -22,6 +22,13 @@ ion-toolbar {
   ion-buttons[slot='secondary'] {
     order: 6;
   }
+
+  * {
```

Comment:
Consider using `@include slotted('*') {...}` as I guess `*` won't work otherwise? E.g. for customTitleTemplate...

Reply:
You're absolutely right about this won't truncate custom titles, however doing @include slotted('*') doesn't ensure that either as the custom title can be anything for instance our example with flex-inline which makes this kind of styling having no effect at all. So the question is if we wanna have any styling like this at our custom titles, or the user of a custom title should take care of these things themself. Otherwise we can ofc. have both as sort of sensible defaults, what do you think?

Reply:
But by using `*` it will be tagged with the host class, whereas `@include slotted('*')` will render as `> *` meaning a direct descendant of the host? Right?